index.html

```diff
@@ -19,8 +19,8 @@ <h2>Thank you for taking part.</h2>
       <label id="emiail-label"  for="email">Email</label>
       <input type="text" placeholder="Enter your email">
       <br>
-      <label id="age-label" for="number" >Age(optinal)</label><br>
-      <input type="number" min="10" max="99" placeholder="Age">
+      <label id="age-label" for="number" >Age</label><br>
```

Comment:
Please add a placeholder for "age" to make it appear on your form as it doesn't show on your form.

Reply:
- [ ] Well done :+1:  you did well but please make sure you highlight the correct area to make the changes and also give out more information about why you are suggesting this make changes.